src/dataprep/main/load_novelty_reuse/dump_db.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
I noted in the current db that the first row contains again the column names. I think, if the csv files has headers, these must be dropped when using the `.import`  in sqlite. (I don't remember exactly how to do it, but I think we're doing that in other files too)